sql/catalyst/src/main/scala/org/apache/spark/sql/internal/StaticSQLConf.scala

```diff
@@ -226,4 +226,11 @@ object StaticSQLConf {
       .version("3.0.0")
       .intConf
       .createWithDefault(100)
+
+  val FOLLOW_HIVE_TABLE_LOCATION_ENABLED =
+    buildStaticConf("spark.sql.follow.hive.table.location")
```

Comment:
Hi, @TJX2014 . Since you are working on these, I'll inform you some community rules. Not every rule is written clearly, so I understand that you may not be aware of that. This is just a tip for your this and next PRs.
1. **NAMING**: `configuration` naming follows a namespace concept. Please don't make a new namespace, `spark.sql.follow` if that is a new set of concept. For all Hive related config, `spark.sql.hive` is used as the namespace. Please see the other existing Hive configurations.
3. **MODULE**: Apache Spark follows modular design. In general, for Hive Specific configurations,  `sql/hive` module is the better place to be considered for a new conf instead of `sql/catalyst` module. Please try to put into `HiveUtils.scala` . If `HiveUtils.scala` is insufficient for your PR, then, you can promote your conf into `sql/core` or `sql/catalyst` modules.